pkgs/applications/blockchains/btcpayserver/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, fetchFromGitHub, fetchurl, nbxplorer, dotnetPackages, dotnetCorePackages, lib }:
+
+let
+
+  deps = import ./deps.nix {inherit fetchurl;};
+  dotnet-sdk = dotnetCorePackages.sdk_3_1;
+
+in
+
+stdenv.mkDerivation rec {
+  name = "btcpayserver";
+  version = "1.0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "btcpayserver";
+    repo = "btcpayserver";
+    rev = "v${version}";
+    sha256 = "1h4567di6sfr6z734b7a7qh0hv5ywq14hlbimxcfxn9hri2g3zbv";
+  };
+  buildInputs = [dotnet-sdk dotnetPackages.Nuget];
+
+  buildPhase = ''
+    mkdir home
+    export HOME=$PWD/home
+
+    export DOTNET_CLI_TELEMETRY_OPTOUT=1
+    export DOTNET_SKIP_FIRST_EXPERIENCE=1
+
+    for package in ${toString deps}; do
+      nuget add $package -Source nixos
+    done
+
+    dotnet restore --source nixos BTCPayServer.Client/BTCPayServer.Client.csproj
+    dotnet build --no-restore -c Release BTCPayServer.Client/BTCPayServer.Client.csproj
+
+    dotnet restore --source nixos BTCPayServer.Data/BTCPayServer.Data.csproj
+    dotnet build --no-restore -c Release BTCPayServer.Data/BTCPayServer.Data.csproj
+
+    dotnet restore --source nixos BTCPayServer.Common/BTCPayServer.Common.csproj
+    dotnet build --no-restore -c Release BTCPayServer.Common/BTCPayServer.Common.csproj
+
+    dotnet restore --source nixos BTCPayServer.Rating/BTCPayServer.Rating.csproj
+    dotnet build --no-restore -c Release BTCPayServer.Rating/BTCPayServer.Rating.csproj
+
+    dotnet restore --source nixos BTCPayServer/BTCPayServer.csproj
+    dotnet build --no-restore -c Release BTCPayServer/BTCPayServer.csproj
+  '';
+
+  installPhase = ''
+      mkdir -p $out/bin/
+      echo -e '#!/usr/bin/env bash \n#launch BTCpayserver \nscriptdir=`dirname $(readlink $(which btcpayserver))` \ndotnet run --no-launch-profile --no-build -c Release -p "$scriptdir/../BTCPayServer/BTCPayServer.csproj" -- $@
+' > $out/bin/run.sh
+      chmod +x $out/bin/run.sh
+      cp -r ./* $out
+      ln -s $out/bin/run.sh $out/bin/btcpayserver
+  '';
+
+  dontStrip = true;
+
+  meta = with lib; {
+    description = "BTCPay Server is a self-hosted, open-source cryptocurrency payment processor. It's secure, private, censorship-resistant and free";
```

Comment:
Description should not [start with a package name](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes).

Reply:
Will push a fix

Reply:
Pushed a fix